From e942846e7aa38f341e76e84844ec2fc1ede63d57 Mon Sep 17 00:00:00 2001
Message-ID: <e942846e7aa38f341e76e84844ec2fc1ede63d57.1701297273.git.sharlatanus@gmail.com>
In-Reply-To: <123a0b9b76c24c29184c527dabb652384f7be762.1701297273.git.sharlatanus@gmail.com>
References: <123a0b9b76c24c29184c527dabb652384f7be762.1701297273.git.sharlatanus@gmail.com>
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Sat, 28 Oct 2023 03:32:46 +0100
Subject: [PATCH v3 13/66] gnu: go-github-com-gorilla-securecookie: Move to
 (gnu packages golang-web).

* gnu/packages/golang.scm (go-github-com-gorilla-securecookie): Move from
here...
* gnu/packages/golang-web.scm: ...to here.

Change-Id: Iae8beed5e7a099033585bcab33cd42d237d2746d
---
 gnu/packages/golang-web.scm | 24 ++++++++++++++++++++++++
 gnu/packages/golang.scm     | 25 -------------------------
 2 files changed, 24 insertions(+), 25 deletions(-)

diff --git a/gnu/packages/golang-web.scm b/gnu/packages/golang-web.scm
index c0ceb5b7ce..1a77e0f6f2 100644
--- a/gnu/packages/golang-web.scm
+++ b/gnu/packages/golang-web.scm
@@ -212,6 +212,30 @@ (define-public go-github-com-gorilla-mux
 incoming requests with their respective handler.")
     (license license:bsd-3)))
 
+(define-public go-github-com-gorilla-securecookie
+  (package
+    (name "go-github-com-gorilla-securecookie")
+    (version "1.1.1")
+    (source
+     (origin
+       (method git-fetch)
+       (uri (git-reference
+             (url "https://github.com/gorilla/securecookie")
+             (commit (string-append "v" version))))
+       (file-name (git-file-name name version))
+       (sha256
+        (base32 "16bqimpxs9vj5n59vm04y04v665l7jh0sddxn787pfafyxcmh410"))))
+    (build-system go-build-system)
+    (arguments
+     '(#:import-path "github.com/gorilla/securecookie"))
+    (home-page "https://github.com/gorilla/securecookie")
+    (synopsis "Encodes and decodes authenticated and optionally encrypted
+cookie values")
+    (description
+     "Gorilla/securecookie encodes and decodes authenticated and optionally
+encrypted cookie values for Go web applications.")
+    (license license:bsd-3)))
+
 (define-public go-github-com-gorilla-websocket
   (package
     (name "go-github-com-gorilla-websocket")
diff --git a/gnu/packages/golang.scm b/gnu/packages/golang.scm
index a3706ebb2e..cab67fea65 100644
--- a/gnu/packages/golang.scm
+++ b/gnu/packages/golang.scm
@@ -4742,31 +4742,6 @@ (define-public go-github-com-andybalholm-brotli
 @url{https://github.com/andybalholm/c2go,https://github.com/andybalholm/c2go}.")
     (license license:expat)))
 
-(define-public go-github-com-gorilla-securecookie
-  (package
-    (name "go-github-com-gorilla-securecookie")
-    (version "1.1.1")
-    (source
-     (origin
-       (method git-fetch)
-       (uri (git-reference
-             (url "https://github.com/gorilla/securecookie")
-             (commit (string-append "v" version))))
-       (file-name (git-file-name name version))
-       (sha256
-        (base32
-         "16bqimpxs9vj5n59vm04y04v665l7jh0sddxn787pfafyxcmh410"))))
-    (build-system go-build-system)
-    (arguments
-     '(#:import-path "github.com/gorilla/securecookie"))
-    (home-page "https://github.com/gorilla/securecookie")
-    (synopsis "Encodes and decodes authenticated and optionally encrypted
-cookie values")
-    (description
-     "Gorilla/securecookie encodes and decodes authenticated and optionally
-encrypted cookie values for Go web applications.")
-    (license license:bsd-3)))
-
 (define-public go-github-com-gorilla-sessions
   (package
     (name "go-github-com-gorilla-sessions")
-- 
2.41.0

